.github/workflows/test.yml

```diff
@@ -207,20 +207,6 @@ jobs:
             tags: bwa/samse
           - profile: conda
             tags: cellpose
-          - profile: conda
-            tags: cellranger/count
```

Comment:
Are we missing `cellranger/count` in the new `test.yml` workflow?

Reply:
No, it was already added there by Adam Talbot ~8 months ago (y)